### ISSUE TITLE
Wrong Summon Tablet Depletion in Cooking

### ISSUE THREAD

**Issue by ghost** [MelvorIdle/melvoridle.github.io#1463]
**Describe the bug**
When cooking, using the Pig + Mole combination, the number of tablets consumed seem to be random, rather than spending 2 pigs and 1 mole

**To Reproduce**
1. Equip Pig and Mole tablets
2. Cook any item (I cooked Fanfish)
3. Check how many tablets depleted per cooking action

**Expected behavior**
Each cooking action should consume 2 Pig tablets and 1 Mole tablet

**Screenshots**
<img width="386" alt="Screen Shot 2021-10-14 at 5 08 35 PM" src="https://user-images.githubusercontent.com/90030253/137292953-d0571cd9-bc2d-4fb7-81c6-fd19180b1425.png">

**Browser**
Firefox

**Comment by SustainableBlueDuck** [comment]
Got the same issue. However **additionally,** another **error** I noticed was that more tablets where used than actually noted by the welcome back screen.
At some point when Cooking Fanfish (and passively cooking) I noticed 8 mole tablets disappearing per cook! I also noticed that this where actually two steps which followed each other very rapidly (initial step of 2 and after 6), this might help indicate the issue?
